circle.yml

```diff
@@ -67,6 +67,11 @@ deployment:
     owner: palantir
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
+  publish-snapshots:
+    branch: /.*/
+    owner: atlasdb
```

Comment:
This is interesting.  Here we see a deployment task targeted at two different build mechanisms (external for docs publish / internal for snapshot publish).  It might be cleaner to consolidate the two, or in as a lighter-weight protection at least at a comment as to who the atlasdb user is (doesn't exist outside of internal builds).